Purchases/Networking/Operations/GetSubscriberDataOperation.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  GetSubscriberDataOperation.swift
+//
+//  Created by Joshua Liebowitz on 11/18/21.
+
+import Foundation
+
+class GetSubscriberDataOperation: NetworkOperation {
+
+    let httpClient: HTTPClient
+    let authHeaders: [String: String]
+    private let customerInfoResponseHandler: CustomerInfoResponseHandler
+    private let customerInfoCallbackCache: CallbackCache<CustomerInfoCallback>
+
+    init(httpClient: HTTPClient,
+         authHeaders: [String: String],
+         customerInfoResponseHandler: CustomerInfoResponseHandler = CustomerInfoResponseHandler(),
+         customerInfoCallbackCache: CallbackCache<CustomerInfoCallback>) {
+        self.httpClient = httpClient
+        self.authHeaders = authHeaders
+        self.customerInfoResponseHandler = customerInfoResponseHandler
+        self.customerInfoCallbackCache = customerInfoCallbackCache
+    }
+
+    func getSubscriberData(appUserID: String, completion: @escaping BackendCustomerInfoResponseHandler) {
```

Comment:
unrelated to this PR (i think)...looking through the code, i'm getting a little confused when subscriberData becomes customerInfo... we have a concept of `subscriberData` within the response from subscribers/appUserId, but we call that response customerInfo, even though the method is getSubscriberData

Reply:
YES right? Confusing. THEY'RE THE SAME THING 🙃  Subscriber/Customer/Purchaser
That's why we did https://github.com/RevenueCat/purchases-ios/issues/876
The backend returns subscriber. So we really probably should finish the rename on the SDK side `s/Subscriber/Customer` and eventually get around to doing that backend-side.

Reply:
ah ok, yeah i think it'd be less confusing if we renamed everything in our sdk, so that only the endpoint string had "subscriber"